spyder/api/widgets/main_widget.py

```diff
@@ -412,11 +412,6 @@ def _setup(self):
         self._options_button.setMenu(self._options_menu)
         self._options_menu.aboutToShow.connect(self._update_actions)
 
-        # Hide icons in Mac plugin menus
```

Comment:
This and the code below are no longer necessary because I centralized the functionality to hide/show icons on Mac menus in `SpyderMenu`.